### ISSUE TITLE
[Bug] NoMetaError: CSF: missing default export (line 1, col 0)  on example stories in typescript vite

### ISSUE THREAD

**Issue by dev-seahouse** [storybookjs/test-runner#261]
### Expected behavior

There should not be any error on example stories generated by storybook.

### Screenshots and/or logs

<img width="1204" alt="image" src="https://user-images.githubusercontent.com/4751342/217204715-0fde9d70-9e88-4f3d-a7e1-97b3532e321c.png">

If applicable, add screenshots and/or logs to help explain your problem.

### Environment
		"@storybook/addon-essentials": "7.0.0-beta.43",
		"@storybook/addon-interactions": "7.0.0-beta.43",
		"@storybook/addon-links": "7.0.0-beta.43",
		"@storybook/blocks": "7.0.0-beta.43",
		"@storybook/react": "7.0.0-beta.43",
		"@storybook/react-vite": "7.0.0-beta.43",
		"@storybook/test-runner": "^0.9.4",
		"@storybook/testing-library": "^0.0.13",
		"@storybook/testing-react": "^1.3.0",
		"storybook": "7.0.0-beta.43",
		"typescript": "^4.9.5",
		"@testing-library/jest-dom": "^5.16.5",
		"@testing-library/react": "^13.4.0",

**Comment by LucidityDesign** [comment]
This is related to [CSF 3.0](https://storybook.js.org/blog/component-story-format-3-0/). The error message (for everyone googling this) is
```
  ● Test suite failed to run

    NoMetaError: CSF: missing default export (line 1, col 0)

    More info: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
```

**Comment by yannbf** [comment]
Thanks for opening this issue! Indeed it was solved in [v0.10.0](https://github.com/storybookjs/test-runner/releases/tag/v0.10.0), so please upgrade if you can <3
